### ISSUE TITLE
Text alignment

### ISSUE THREAD

**Issue by earion68** [pkissling/clock-weather-card#189]
**Is your feature request related to a problem? Please describe.**
Current conditions are right-aligned (above the time) and today's date is left-aligned (below the time).

**Describe the solution you'd like**
Ideally, we should be able to mention the alignment we like for forecast, time and date, individually, with a specific card configuration.
Example:
```
  date-align: center
  time-align: center
  current-conditions-align: left
```

**Describe alternatives you've considered**
card-mod and CSS changes but I'm too bad at CSS to achieve what I want.

**Comment by grolnn** [comment]
I agree. The slight misalignment of the date compared to the alignment of the time is driving me nuts! :)

**Comment by dani315p00n** [comment]
I would also like the option define the alignment of these so that they can all be centered.

**Comment by c-kick** [comment]
For those struggling with the correct css to achieve this with card_mod, use this to align everything to the **right**:

```
card_mod:
  style: |
      clock-weather-card-today-right,
      clock-weather-card-today-right-wrap > * {
        text-align: end !important;
        justify-content: flex-end !important;
      }
```

Change `end` and `flex-end` to `start` and `flex-start` respectively, to have everything line out to the **left**.
